biolink-model.yaml

```diff
@@ -6998,6 +7168,38 @@ classes:
     is_a: named thing
     abstract: true
 
+  supporting study result set:
+    abstract: true
+    is_a: information content entity
+    description: >-
+      A result of a supporting study.
```

Comment:
not sure what a 'supporting study result set' is . . . the definition here doesn't help.

Reply:
done!